inc/cachify.settings.php

```diff
@@ -70,6 +70,7 @@
 					<label for="cachify_reset_on_comment">
 						<input type="checkbox" name="cachify[reset_on_comment]" id="cachify_reset_on_comment" value="1" <?php checked( '1', $options['reset_on_comment'] ); ?> />
 						<?php esc_html_e( 'Flush the cache at new comments', 'cachify' ); ?>
+						<?php esc_html_e( '(Only works for comments which comes from the front-end, not added from the back-end)', 'cachify' ); ?>
```

Comment:
The singular verb comes does not appear to agree the plural subject comments, I mean it should use come.

Reply:
Small mindset. Maybe we should also add a point after each sentence. However currently we it not always on the messages.